### ISSUE TITLE
Navigate to specific visibleItems using groupKey

### ISSUE THREAD

**Issue by papakojo** [naver/egjs-infinitegrid#395]
## Description
Hey, great library!

I'm using the angular version with the justified layout and was wondering if there is a way you can click on a button to navigate to a specific set visibleItems. I noticed you have a `moveTo(index, itemIndex)` method in your API but I can't locate it in any of the element review in the console.

If there isn't any method for it, can you point in a direction where it can be easily done? 
thx!

**Comment by droplab** [comment]
HI, i am wondering the same exact thing. Thanks for the work on a great library

**Comment by daybrush** [comment]
@papakojo 

![image](https://user-images.githubusercontent.com/3433062/116650663-9e513500-a9bc-11eb-985f-b4cf2c343c63.png)


There is a moveTo method, but what is the ngx-infinitegrid version?

Would you like to update to the latest version?

**Comment by papakojo** [comment]
@daybrush Thx! `moveTo` works well. In angular, my `ViewChild` type was set to `any` instead of `NgxGridLayoutComponent`. 

It looks like the only way to navigate to a specific group using `moveTo` is if the `groupKey` is loaded in the existing `items` array. So I was trying to preload the `items` array before the component initializes, so I can move around easily. I've tried different ways that work but I noticed that the `visibleItems` array also preloads the same data and everything gets rendered in the DOM.
Is there a way to preload everything into the `items` array and when `onAppend` or `moveTo` it shows just the 20/40 `visibleItems` using groupKey?
